source/includes/api/_areas_v2.md

```diff
@@ -221,25 +289,24 @@ curl -X POST https://api.resourcewatch.org/v2/area
-language             | The language of this area. Defaults to 'en'.                                                       | String  | 'es'       |
-geostore             | An ID of a geostore to which this area relates to.                                                 | String  | '123'      |
-public               | If the area is public or not. Defaults to false.                                                   | Boolean | true       |
-fireAlerts           | If the area is intended to subscribe to fire alerts. Defaults to false.                            | Boolean | true       |
-deforestationAlerts  | If the area is intended to subscribe to deforestation alerts. Defaults to false.                   | Boolean | true       |
-monthlySummary       | If the area is intended to subscribe to monthly summaries. Defaults to false.                      | Boolean | true       |
-email                | Email to be provided to the subscription.                                                          | String  | youremail@resourcewatch.org |
-webhookUrl           | Webhook URL to be provided to the subscription (only used in case the email is not set).           | String  | https://www.google.com/ |
-status               | The status of the area - either 'saved' or 'pending'. Read-only attribute.                         | String  | 'saved'    |
-subscriptionId       | The ID of the subscription associated with this area. Read-only attribute.                         | String  | 5e4d7c47dd8fa31290d548ae |
+Use this endpoint to create new areas. For a detailed description of each field that can be provided in the body of the request, check out the [Area reference](#area-reference) section.
+
+Keep in mind that you should provide one of the following when creating an area:
+
+* `geostore` with the ID of a geostore object if you are creating an area that references a geostore;
+* `wdpaid` with the ID of a protected area if you are creating an area that references a protected area;
+* `iso` object with a valid country/region/subregion if you are creating an area that references an admin country/region/subregion;
+* `use` object with valid id and name of a land use concessioned area, if you are creating an area that references a land use area.
+
+**This endpoint requires authentication, and it will return 401 Unauthorized if none is provided.**
+
+### Errors for creating an area
+
+Error code     | Error message (example)     | Description
+-------------- | --------------------------- | ------------------------------------------------------------
+400            | `<field> can not be empty.` | You are missing a required field while creating the area.
+400            | `<field> is not valid.`     | You provided an invalid field while creating the area.
+401            | `Unauthorized`              | No token was provided.
 
 ### Email notification
 
```

Comment:
I would move this "According to multiple factors..." paragraph to the end of the previous section